### ISSUE TITLE
Fast Insert

### ISSUE THREAD

**Issue by Kencho5** [rhsimplex/image-match#151]
as you mentioned in the readme, it states that it can easily scale up to billions of images, but i'm having a hard time inserting a lot of images in elasticsearch. i've tried to do it with multiprocessing, threading and even async too, but given that i have around 160 million images it's still slow. any suggestions on how i can insert those images a bit faster?

**Comment by trashmit** [comment]
did you come up with a solution?

**Comment by Kencho5** [comment]
yes. made bulk insert script, but don't have it anymore. you have to split functions and use multiprocessing
